portal/server/api.js

```diff
@@ -884,6 +884,27 @@ const uploadUtilisationReportData = async (uploadingUser, reportPeriod, csvData,
   }
 };
 
+/**
+ * Generates validation errors for all errors in the utilisation report data
+ * @param {import('@ukef/dtfs2-common').UtilisationReportCsvRowData[]} reportData - The csv data with location information
+ * @param {string} userToken - The user token
+ * @returns {Promise<{csvValidationErrors: import('@ukef/dtfs2-common').UtilisationReportDataValidationError[]}>} - The validation errors for the utilisation report data
```

Comment:
Can we have a simple return type instaed of `import('...`?

Reply:
Sorry I'm not sure I understand what alternative way there is to type this?

Reply:
Same reply as here => https://github.com/UK-Export-Finance/dtfs2/pull/3413#discussion_r1741064160